thoth-api/src/model/title/mod.rs

```diff
@@ -0,0 +1,169 @@
+use crate::model::locale::LocaleCode;
+use diesel::prelude::*;
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+use crate::graphql::utils::Direction;
+use crate::model::{HistoryEntry};
+use crate::schema::title;
+use crate::schema::title_history;
+
+#[cfg_attr(
+    feature = "backend",
+    derive(juniper::GraphQLEnum),
+    graphql(description = "Field to use when sorting fundings list")
+)]
+pub enum TitleField {
+    TitleId,
+    WorkId,
+    FullTitle,
+    Title,
+    Subtitle,
+    Canonical,
+    LocaleCode,
+}
+
+#[cfg_attr(
+    feature = "backend",
+    derive(juniper::GraphQLInputObject),
+    graphql(description = "Field and order to use when sorting affiliations list")
+)]
+pub struct TitleOrderBy {
+    pub field: TitleField,
+    pub direction: Direction,
+}
+
+impl Default for TitleOrderBy {
+    fn default() -> Self {
+        Self {
```

Comment:
I think the default one in this case can be `TitleField:Canonical` with `Direction:DESC`, so the canonical title comes first by default